treescript/src/treescript/git.py

```diff
@@ -176,9 +176,10 @@ async def push(config, task, repo_path, target_repo):
     target_repo_ssh = extract_github_repo_ssh_url(target_repo)
     repo.remote().set_url(target_repo_ssh, push=True)
     log.debug("Push using ssh command: {}".format(git_ssh_cmd))
+    branch = get_branch(task, "master")
```

Comment:
Nit, not blocking for now, question for later - will this fail when the mobile projects switch their default branch to `main` ?

Reply:
That's right. The git protocol exposes a way for client to know what the default branch is. It would be a matter of porting https://hg.mozilla.org/ci/ci-admin/rev/c6cc6365b80433445b5c70047aa50e0952d7407c to treescript.